openapi/ga4gh-tool-discovery.yaml

```diff
@@ -673,29 +682,66 @@ definitions:
+          resolve to the raw unwrapped content that would otherwise be available
+          in content. One of url or content is required.
         example:
           descriptorfile:
-            url: https://raw.githubusercontent.com/ICGC-TCGA-PanCancer/pcawg_delly_workflow/ea2a5db69bd20a42976838790bc29294df3af02b/delly_docker/Delly.cwl
+            url: >-
+              https://raw.githubusercontent.com/ICGC-TCGA-PanCancer/pcawg_delly_workflow/ea2a5db69bd20a42976838790bc29294df3af02b/delly_docker/Delly.cwl
           containerfile:
-            url: https://raw.githubusercontent.com/ICGC-TCGA-PanCancer/pcawg_delly_workflow/c83478829802b4d36374870843821abe1b625a71/delly_docker/Dockerfile
+            url: >-
+              https://raw.githubusercontent.com/ICGC-TCGA-PanCancer/pcawg_delly_workflow/c83478829802b4d36374870843821abe1b625a71/delly_docker/Dockerfile
   ServiceInfo:
     type: object
-    description: Describes this registry to better allow for mirroring, indexing, and useful information about the running service, including supported versions and
-      default settings.
+    description: >-
+      Describes this registry to better allow for mirroring, indexing, and
+      useful information about the running service, including supported versions
+      and default settings. Partially synced with
+      https://github.com/ga4gh-discovery/service-info
     required:
+      - id
+      - name
       - version
       - api_version
       - contact_info_url
     properties:
+      id:
+        type: string
+        description: >-
+          Unique ID of this service. Reverse domain name notation is
+          recommended, though not required.
+        example: org.ga4gh.service
+      name:
+        type: string
+        description: Name of this specific service.
+        example: 1000 Genomes Project
+      description:
+        type: string
+        description: Description of the service.
+        example: >-
+          The 1000 Genomes Project is the largest public catalogue of human
+          variation and genotype data.
+      documentationUrl:
+        type: string
+        description: URL of the documentation of this service (RFC 3986 format).
+        example: 'https://docs.example.com'
+      contactUrl:
+        type: string
+        description: >-
+          URL of the contact for the host/maintainer of this service, e.g. a
+          link to a contact form (RFC 3986 format), or an email (RFC 2368
+          format). Users of the endpoint should use this to report problems or
+          security vulnerabilities.    
+        example: 'mailto:support@example.com'
       version:
         type: string
-        description: The version of this registry
+        description: Version of the service.
+        example: '0.1'
       api_version:
         type: string
         description: The version of the GA4GH tool-registry API supported by this registry
+        example: v2
```

Comment:
Beyond the scope of this PR, but since I've never looked at this before, I'll ask here: shouldn't api_version be an array? Dockstore supports both v1 and v2.

Reply:
For background, I'm going to interpret this a bit. 

I think this is the right place for minor versions in an array (e.g. 2.1, 2.2) but v1 implementations should be at an entirely different basePath https://github.com/ga4gh/tool-registry-service-schemas/blob/95bedc7f525cb1a85d25c6974bb6a9e9984086f1/openapi/ga4gh-tool-discovery.yaml#L21

So I think I'll refine the language a bit to indicate the versions supported by the endpoints at this basePath